docker-compose.yml

```diff
@@ -9,6 +9,8 @@ services:
       - "8000:8000"
     volumes:
       - ./frontend/src/:/opt/frontend/src/
+      - ./frontend/.eslintrc.js:/opt/frontend/.eslintrc.js
```

Comment:
I'm not sure this is the solution. I think the solution should be to simply create another docker image and push to registry, since the new image will contain the correct linting files. What do you think?